src/Generator.cpp

```diff
@@ -21,362 +26,655 @@
 #include "HGCalSimulation/FastShower/interface/Event.h"
 #endif
 
+using namespace std;
+
+Generator::Generator(const Parameters& params): geometry_(params.geometry()),
+                                                output_(params.general().output_file),
+                                                shower_(params.shower()),
+                                                parameters_(params) { 
+}
+
+
+Generator::~Generator(){
+}
+
+
+double** Generator::readCalibration(const string filename) {
+
+    double** calib = new double*[52];
+    for(int i = 0; i < 52; ++i)
```

Comment:
Avoid magic numbers. Use a constant variable for this 52.

Reply:
Fixed. Now the magic number are declared in Parameters.h